dockerfiles/archlinuxarm/build.sh

```diff
@@ -4,12 +4,21 @@
 
 URL="http://os.archlinuxarm.org/os"
 IMAGE_TAR="ArchLinuxARM-rpi-2-latest.tar.gz"
+TODAY="$(date '+%Y-%m-%d')"
 
-if [ ! -f ${IMAGE_TAR} ]; then
-    wget "${URL}/${IMAGE_TAR}"
-fi
-
+# Download and unpack the base system tarball from archlinuxarm.org.
+rm -rf "${IMAGE_TAR}" arch-rootfs
+wget "${URL}/${IMAGE_TAR}"
 mkdir -p arch-rootfs
 tar -xzf ${IMAGE_TAR} --strip-components=1 --directory=./arch-rootfs
 
-docker build -f Dockerfile -t "iterait/archlinuxarm:$(date '+%Y-%m-%d')" -t "iterait/archlinuxarm:latest" --squash .
+# Build the base image.
+docker build -f archlinuxarm -t "iterait/archlinuxarm:${TODAY}" -t "iterait/archlinuxarm:latest" --squash .
+# Build the -dev version without context.
+docker build - -t "iterait/archlinuxarm-dev:${TODAY}" -t "iterait/archlinuxarm-dev:latest" < archlinuxarm-dev
+
+# Push the images to the repository.
+docker push iterait/archlinuxarm:latest
+docker push iterait/archlinuxarm:${TODAY}
+docker push iterait/archlinuxarm-dev:latest
+docker push iterait/archlinuxarm-dev:${TODAY}
```

Comment:
Do we actually need to push the images? It should be done automatically...

Reply:
However, the image needs to be built on ARM device. Therefore, we build and push it manually on one of our ARM devices whenever needed.